core/shared/src/test/scala/me/shadaj/scalapy/py/SpecialSyntaxTest.scala

```diff
@@ -23,6 +32,15 @@ class SpecialSyntaxTest extends AnyFunSuite {
     }
   }
 
+  test("Can access and update the list elements using brackets") {
+    local{
```

Comment:
Nit, need a space between `local` and `{`, and remove the extra space between `val` and `myList`.